etc/bash_completion.d/dnf

```diff
@@ -20,12 +20,60 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
 # 02110-1301  USA
 
-_dnf_helper()
+__dnf_main_options="-4 -6 --advisory --advisories --allowerasing --assumeno
```

Comment:
To have a list of all options has one disadvantage, any change in dnf options could result in outdated list.

Reply:
Do you have any suggestions on how to improve this? I tried to think of a better solution, but couldn't come up with anything.

Reply:
I can only redirect to other bugzilla where one suggestion was make https://bugzilla.redhat.com/show_bug.cgi?id=1172475 . How it can help I cannot tell.